src/main/java/sirius/kernel/xml/AbstractStructuredOutput.java

```diff
@@ -275,4 +275,12 @@ public StructuredOutput nullsafeProperty(@Nonnull String name, @Nullable Object
         property(name, data != null ? data : "");
         return this;
     }
+
+    @Override
+    public StructuredOutput propertyIfFilled(@Nonnull String name, @Nullable Object data) {
+        if (data != null) {
```

Comment:
whats the point of that? using property: null is more obvious and should yield the same result